src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1989,16 +1989,19 @@ jint os::init_2(void) {
     if (status != 0) {
       log_info(os)("os::init_2 getrlimit failed: %s", os::strerror(errno));
     } else {
+      // On macOS >= 10.6 if we define _DARWIN_UNLIMITED_STREAMS or _DARWIN_C_SOURCE
+      // (we define _DARWIN_C_SOURCE) we can ask for RLIM_INFINITY
```

Comment:
You could fold this information into the comment block below so that it is within the ifdef for APPLE.

Reply:
Done.